### PR TITLE
feat(plp): category slider using builder io

### PR DESCRIPTION
**Pull request by kibo-geetanshuchhabra**

### CODE REVIEW

File: lib/helpers/buildCategorySearchParams.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import type { CategoryPagedCollectionParams } from '../types'
+import type { QueryCategoriesArgs } from '@/lib/gql/types'
+
+export const buildCategorySearchParams = ({
+  pageSize = 16,
```
Comment (kibo-sushant) [inline lib/helpers/buildCategorySearchParams.ts]: wouldn't it be nice if will make it configurable?

**Review by kibo-sushant — APPROVED** [review]

File: components/product-listing/CategorySlider/CategorySlider.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  const { categoryCodes } = props
+  const { getCategoryLink } = uiHelpers()
+  const { data: categorySearchResult } = useCategorySearchQueries(categoryCodes)
+  const categories = categorySearchResult?.items
+  const breakPoints = [
+    { width: 1, itemsToShow: 1 },
+    { width: 550, itemsToShow: 2 },
+    { width: 768, itemsToShow: 4 },
+    { width: 1200, itemsToShow: 5 },
+  ]
```
Comment (TriptiVermaXP) [inline components/product-listing/CategorySlider/CategorySlider.tsx]: try to make it configurable.

**Review by TriptiVermaXP — APPROVED** [review]
Approving, please check small comment.

File: components/product-listing/CategorySlider/CategorySlider.tsx (hunk 2 above)
Comment (kibo-geetanshuchhabra) [reply components/product-listing/CategorySlider/CategorySlider.tsx]: Resolved

File: lib/helpers/buildCategorySearchParams.ts (hunk 1 above)
Comment (kibo-geetanshuchhabra) [reply lib/helpers/buildCategorySearchParams.ts]: Resolved

**Review by kibo-kevinwatts — APPROVED** [review]

**Pull request merged**